DOC-1871: RC - SAML Integration guide migration (#2345)
RedisLabs/redislabs-docs: content/rc/security/single-sign-on/saml-sso/saml-integration-aws-identity-center.md

2. You will then be able to **download** the service provider (SP) metadata. Save the file to your local hard disk.

   {{<image filename="images/rc/saml/aws_iam_identity_center_saml_9.png" alt="" >}}{{</image>}}

3. Open the file in any text editor, and there are 2 pieces of information that you need to copy and mark down:

* **EntityID** : The unique name of the service provider (SP)

  {{<image filename="images/rc/saml/sm_saml_4.png" alt="" >}}{{</image>}}

* **Location** : The location of the assertion consumer service

  {{<image filename="images/rc/saml/sm_saml_5.png" alt="" >}}{{</image>}}

## Step 3 - Finish SAML configuration in AWS IAM Identity Center's Redis Cloud Application

1. Go back to the **Configuration setup** screen in IAM identity Center. Scroll down to the bottom of the page and select the **Upload application SAML metadata file** option. Click on the **upload** button and choose the file that you downloaded in the SAML configuration screen in Redis Cloud. It should look like the screen below :

{{<image filename="images/rc/saml/aws_iam_identity_center_saml_10.png" alt="" >}}{{</image>}}

> 2. If you would like to also configure an IdP initiated workflow, then you need to fill in the **relay state** field in the **Application properties** section. Use the following URL **https://app.redislabs.com/#/login/?idpId=XXXXXX**. Take the ID from the location URL in step 3 (the content after the last forward slash "/") and append to the url. It should look like the screen below :

Reviewer: At the end of the sentence, "url" should be capitalized as "URL".

{{<image filename="images/rc/saml/aws_iam_identity_center_saml_11.png" alt="" >}}{{</image>}}

> 3. Once all the information has been filled in, it is time finish the application creation by clicking the **Submit** button.

Reviewer: Our style guides recommend using "after" instead of "once".

{{<image filename="images/rc/saml/aws_iam_identity_center_saml_12.png" alt="" >}}{{</image>}}

4. Next, we need to configure the **Redis Cloud** application's attribute mappings. Click on the **Actions** menu and choose **Edit Attribute Mappings**. 

{{<image filename="images/rc/saml/aws_iam_identity_center_saml_13.png" alt="" >}}{{</image>}}   

In the next screen, add the following attributes :

* Subject : **${user:email}**, **unspecified**
* Email : **${user:email}**, **unspecified**
* FirstName : **${user:givenName}**, **unspecified**
* LastName : **${user:familyName}**, **unspecified**
* redisAccountMapping : **XXXXXXX=owner**, **unspecified**

> The **redisAccountMapping** key-value pair consists of the **lower-cased role name** (ie owner, member, manager, or viewer) AND your **Redis Cloud Account ID** (you can find this information in the upper-right user menu at app.redislabs.com).

Reviewer 2: ```suggestion
The **redisAccountMapping** key-value pair consists of the **lower-cased role name** (owner, member, manager, or viewer) AND your **Redis Cloud Account ID** found in the [account settings]({{<relref "rc/accounts/account-settings">}}).
```

1. The **Configure Application** screen is where you initially get the information needed to configure SAML in Redis Cloud. To begin, change the **Display name** and **Description** to **Redis Cloud**.

   {{<image filename="images/rc/saml/aws_iam_identity_center_saml_5.png" alt="" >}}{{</image>}}

1. Next, scroll to the **IAM Identity Center metadata** section. Here, you will find all of the information needed to configure SAML in Redis Cloud:

* IAM Identity Center sign-in URL
* IAM Identity Center SAML issuer URL
* IAM Identity Center Certificate

Note down or copy the URLs and select **Download** to download the certification information.

{{< note >}}
Both the IAM Identity Center sign-in URL and the IAM Identity Center SAML issuer URL are the same value. This is expected.
{{< /note >}}

   {{<image filename="images/rc/saml/aws_iam_identity_center_saml_6.png" alt="" >}}{{</image>}}

## Step 2: Configure SAML support in Redis Cloud

Now that you have your IAM Identity Center IdP server information, configure support for SAML in Redis Cloud.

### Log in to your Redis Cloud SM account

Log in to your SM account at [Redis admin console](https://app.redislabs.com/#/login)

> ### Activate SAML in access management

Reviewer 3: ```suggestion
### Activate SAML in Access Management
```
